batch/batch/front_end/front_end.py

```diff
@@ -966,7 +965,7 @@ async def _create_jobs(
                         reason=f'Cloudfuse requests with mount_path=/io are not supported. Found {config}'
                     )
 
-        if spec.get('mount_tokens', False):
+        if spec.get('mount_tokens', False) and not os.environ.get('HAIL_TERRA', False):
```

Comment:
should this be an assertion instead? It seems wrong to ask to mount tokens if we don't have them.

Reply:
True I'll adjust the client not to request them